codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -646,9 +647,10 @@ private LastHttpContent readTrailingHeaders(ByteBuf buffer) {
                         b.append(currentLastPos)
                          .append(lineTrimmed);
                         current.set(lastPos, b.toString());
-                    } else {
-                        // Content-Length, Transfer-Encoding, or Trailer
                     }
+                    //else {
+                    // Content-Length, Transfer-Encoding, or Trailer
```

Comment:
I don't remember why I added that comment anymore unfortunately. I usually add a comment to an empty `else` block to explain when the condition of `if` clause will not match.